include/lcp-paginator.php

```diff
@@ -37,23 +37,26 @@ public function get_pagination($params){
       $this->next_page_num = null;
       if ($pages_count > 1){
           for($i = 1; $i <= $pages_count; $i++){
-              $lcp_paginator .=  $this->lcp_page_link($i, $params['page'], $params['instance']);
+              $lcp_paginator .=  $this->lcp_page_link($i, $params['page'], $params['instance'],
+                                                      null, $params['bookmarks']);
```

Comment:
Yeah, absolutely, that `null` doesn't look good in there. Just drop a short note in the changelog that this method has been refactored in case some users customized stuff in this file.